x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/outlier_exploration/outlier_exploration.tsx

```diff
@@ -90,6 +91,8 @@ export const OutlierExploration: FC<ExplorationProps> = React.memo(({ jobId }) =
     jobConfig?.analyzed_fields?.excludes,
     resultsField
   );
+  const destIndex =
```

Comment:
This logic is used in several places in DFA like `useResultsViewConfig` or `deleteAnalyticsAndDestIndex`... Might be out of scope of this PR but it's better to have a utility function (e.g. `getDestinationIndex`) so future updates to the logic can be more consistent.

Reply:
Added util function in c849b8047a8c99e304f691680d81ca8d56eb0492